yt/geometry/grid_container.pyx

```diff
@@ -55,12 +55,23 @@ cdef class GridTree:
                   np.ndarray[np.float64_t, ndim=2] left_edge,
                   np.ndarray[np.float64_t, ndim=2] right_edge,
                   np.ndarray[np.int32_t, ndim=2] dimensions,
-                  np.ndarray[np.int64_t, ndim=1] parent_ind,
+                  list parent_ind, # Not an array anymore!
```

Comment:
This comment will probably not age well. Also, I think Cython 3 supports more expressive uses of Python type annotations like `list[list[float]]`, might be worth a try
```suggestion
                  list parent_ind,
```

Reply:
I didn't know that -- I'll give that a go.